emails/views.py

```diff
@@ -705,49 +712,42 @@ def _strip_localpart_tag(address):
 
 
 def _build_reply_requires_premium_email(
-    from_address,
-    reply_record,
-    message_id,
-    decrypted_metadata,
-):
-    # If we haven't forwaded a first reply for this user yet, _reply_allowed
+    from_address: str,
+    reply_record: Reply,
+    message_id: str | None,
+    decrypted_metadata: dict[str, Any] | None,
+) -> MIMEMultipart:
+    # If we haven't forwarded a first reply for this user yet, _reply_allowed
     # will forward.  So, tell the user we forwarded it.
     forwarded = not reply_record.address.user.profile.forwarded_first_reply
-    sender = ""
+    sender: str | None = None
```

Comment:
They aren't quite equivalent... if you pass `ctx = {"sender": None}`, then you'll get an FTL error:

```
FTL exception for locale [], message 'first-reply-forwarded', args {'sender': None}: TypeError("Unsupported external type: sender, <class 'NoneType'>")
```

but it can be set to `""` here, since we protect against passing `None` a few lines down, and that will make it clearer that we're adding types.